### PR TITLE
Make use of Rails I18n data format in EventSerializer #598

### PR DESCRIPTION
**Pull request by Shriyanshagro**
Internationalized data format , Can be used in future , handling configuration for different locales.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -40,6 +40,12 @@ gem 'rolify'
 # to show flash messages from ajax requests
 gem 'unobtrusive_flash', '>=3'
 
+
+# Internationalizing gem , use gem according to version of your rails application
+# gem 'rails-i18n', '~> 5.0.0.beta1' # For 5.0.0.beta1
+gem 'rails-i18n', '~> 4.0.0' # For 4.0.x
+# gem 'rails-i18n', '~> 3.0.0' # For 3.x
```
Comment (hennevogel) [inline Gemfile]: We know which rails version we use don't we? :-)

Comment (Shriyanshagro) [inline Gemfile]: Yeah we know, I just comment it out in case if we ever switch to higher version.
So should I remove commented lines ??

Comment (hennevogel) [inline Gemfile]: Yes you should

Comment (Shriyanshagro) [inline Gemfile]: task done :-)

**Comment by hennevogel** [comment]
respect_the_rubicop

**Pull request closed without merging**